.github/workflows/vulnerability-monitor.yaml

```diff
@@ -0,0 +1,37 @@
+name: Vulnerability-Monitor
+
+on:
+  schedule:
+    - cron: "0 16 * * *"
+
+jobs:
+  docker-image-monitor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - run: |
+          sudo apt-get update
```

Comment:
Why not https://github.com/docker/scout-action ?

Reply:
I tried it on my local, and it wasnt working, stuck on setting up the scout-action while trying to clone the repo due to binaries stored in repo https://github.com/docker/scout-action/tree/main/dist.

Reply:
@ekampf, @emrul  and I looked at this in more details and we think there is also a security consideration. The scout action includes a binary and we cannot see the source code for this binary, see this commit as an example https://github.com/docker/scout-action/commit/80fc6e99dd73c7d09db18aa7d4006bebb3f5c11a#diff-a74d462be054c5a1a941b83a506392c5c7caf269aeea819f1d7d13c483a41f93

Reply:
Its the **official docker action** not some binary from a random person on the internet